pkg/applicationserver/io/packages/alcsync/v1/alcsync.go

```diff
@@ -60,27 +65,58 @@ func newTimeSyncCommand(
 }
 
 // MakeCommands parses the uplink payload and returns the commands.
-func MakeCommands(up *ttnpb.ApplicationUplink, fPort uint32, data *packageData) ([]Command, error) {
-	cID, cPayload := up.FrmPayload[0], up.FrmPayload[1:]
+func MakeCommands(up *ttnpb.ApplicationUplink, fPort uint32, data *packageData) ([]Command, []events.Builder, error) {
+	cIDByte, cPayload := up.FrmPayload[0], up.FrmPayload[1:]
```

Comment:
You can also cast here directly, and same below, if you want to avoid having two notations (`cIDByte` and `cID`).